src/layouts/HomePageLayout.js

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  GlobalHeader,
+  MobileHeader,
+  LoggedInProvider,
+} from '@newrelic/gatsby-theme-newrelic';
+import { isNavClosed, setNavClosed } from '../utils/navState';
+import { css } from '@emotion/react';
+import { scroller } from 'react-scroll';
+import { CSSTransition } from 'react-transition-group';
+import Layout from '../components/Layout';
+import SEO from '../components/SEO';
+import RootNavigation from '../components/RootNavigation';
+import { useLocation } from '@reach/router';
+import { MainLayoutContext } from '../components/MainLayoutContext';
+
+const MainLayout = ({ children, pageContext }) => {
```

Comment:
Do we still have a `MainLayout` template? If so let's rename this to be `HomePageLayout` and export it as such.